src/main/kotlin/pt/up/fe/ni/website/backend/dto/entity/ProjectDto.kt

```diff
@@ -5,5 +5,6 @@ import pt.up.fe.ni.website.backend.model.Project
 class ProjectDto(
     val title: String,
     val description: String,
-    val isArchived: Boolean = false
+    val isArchived: Boolean = false,
+    val technologies: List<String>
```

Comment:
```suggestion
    val technologies: List<String> = emptyList()
```
It doesn't need to be required in the dto, just default to empty